plugins/kubernetes/app/models/kubernetes/api/pod.rb

```diff
@@ -5,17 +5,39 @@ def initialize(api_pod)
         @pod = api_pod
       end
 
-      def ready?
-        @pod.status.phase == 'Running' && condition_ready?
-      end
-
       def name
         @pod.metadata.name
       end
 
-      private
+      def live?
+        @pod.status.phase == 'Running' && ready?
+      end
 
-      def condition_ready?
+      def phase
+        @pod.status.phase
+      end
+
+      def project_id
+        @pod.metadata.labels.project_id.to_i
+      end
+
+      def release_id
+        @pod.metadata.labels.release_id.to_i
+      end
+
+      def deploy_group_id
+        @pod.metadata.labels.deploy_group_id.to_i
+      end
+
+      def role_id
+        @pod.metadata.labels.role_id.to_i
+      end
```

Comment:
Good point.

Personally, I tend to avoid using metaprogramming for this things, because you loose the IDE assistance when you do it, but it appears to be a common approach in Ruby, so... I'll do it. ;)